user-guide/Advanced_Modules/SRM/Advanced_SRM_settings.md

```diff
@@ -2,24 +2,23 @@
 uid: Advanced_SRM_settings
 ---
 
-# Advanced SRM settings
+# Function resources
 
-From DataMiner 9.5.7 onwards, advanced SRM settings can be configured in the file *Config.xml*, in the folder *C:\\Skyline DataMiner\\ProtocolFunctionManager*. This file contains the following tags:
+From DataMiner 9.5.7 onwards, some advanced settings related to function DVEs linked with a function resource, can be configured in the file *Config.xml*, in the folder *C:\\Skyline DataMiner\\ProtocolFunctionManager*. This file contains the following tags:
 
 | Tag | Description |
 |--|--|
-| ActiveFunctionResourcesThreshold | Used to configure the number of virtual function resources that can be active at the same time before a disabling of virtual function resources is triggered. Set this to an integer representing the maximum number of active virtual function resources. By default: 100. |
-| FunctionHysteresis | Can be used to keep virtual function resources from being disabled if they will be used again soon. Set this to the number of seconds in the future that the system has to check whether a virtual function resource is in use, in order to determine whether it should be disabled. By default, up to DataMiner 9.5.12, this is set to *PT0S*, which means no hysteresis is applied. From DataMiner 9.5.13 onwards, the default value is 10 minutes. |
-
-> [!NOTE]
-> From DataMiner 9.5.8 onwards, the hysteresis functionality is expanded to also include the preloading of virtual function resources. This means that if, for example, function hysteresis is set to 10 minutes, any disabled virtual function resources that are required for a booking instance will be enabled 10 minutes before the start of the booking instance.
+| ActiveFunctionResourcesThreshold | Used to configure the number of function DVEs that can be active at the same time. Their deactivation is scheduled after a booking has finished. They only get deactivated if the threshold is reached. The default threshold is 100. <br/> The threshold will not act as a limit, since it is not considered during the activation of function DVEs. |
```

Comment:
Instead of the last sentence maybe something like this would be more clear?
"DVEs currently in use by bookings or scheduled for use within the hysteresis time will not be automatically disabled."